Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,50 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the **users** group (which has a GID of 100).
+
+This program must be run as root.
+
+## Options
+
+* `-u`, `--uid` _uid_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
+* `-g`, `--gid` _gid_: The group identifier for the new user. If not specified, it will default to 100 (the **users** group).
+* `-s`, `--shell` _shell_: The shell binary for this login. The default is `/bin/Shell`. 
+* `-m`, `--create-home`: Create a home directory for this new user at `/home/username`.
+* `-d`, `--home-dir` _path_: Set the home directory for this user to path).
```

Comment:
stray `)`

And missing "By default, ..." (this is where it should say about `/home/username`, not above)

Reply:
`Set the home directory for this user to path). By default, this is `/home/username`, where `username` is the value of `login`.`??